designs/administrator/olm/catalogsource-as-providertype/index.md

```diff
@@ -0,0 +1,26 @@
+---
+parent: Administrator
+version: 4.5
+---
+
+# Custom CatalogSource as OperatorHub ‘Provider Type’ filter facet
+
+When a custom CatalogSource is added today, it appears in the “Custom” facet in the ‘Provider Type’ filter in OperatorHub. Customers are interested in having these CatalogSources show up as their own facet instead of being lumped under “Custom.”
+
+A new annotation will be added to CatalogSources that allows the creator to specify a display-name of that CatalogSource. This value will appear as the facet in ‘Provider Type’ filter. If a display-name isn’t specified, the actual CatalogSource Name will be used.
+
+The "Custom" provider type will no longer exist going forward, and is instead replaced by each CatalogSource being surfaced uniquely. 
+
+## OperatorHub ‘Provider Type’ filter
+
+![OperatorHub Provider Type filter](img/1-1-OH.png)
+- Custom catalog sources have been added to the cluster and appear in the **Provider Type** filter.
+- “Custom” provider type no longer exists, instead each CatalogSource is shown individually.
+- One custom source, “Our Operators,” has a display name specified.
+- The other custom source, “console3-test-operators,” does not have a display name specified and the CatalogSource name is shown.
+- Ideally **Red Hat**, **Certified**, **Community** and **Marketplace** facets would appear first, with the rest alphabetized after, though all alphabetized is also acceptable.
+- If more than 6 Provider Type facets appear, they will be surfaced under an expandable **Show # more** link, similar to Provider filter.
+- Names are truncated as needed.
```

Comment:
Does the full name show on hover or click?

Reply:
Good catch! The items in the sidebar of OperatorHub don't get truncated today, they just start wrapping as shown below. Do we think this should be changed, or is the current behavior okay? I wonder how often custom CatalogSources will have very long names.

![image](https://user-images.githubusercontent.com/9122899/77559816-06b51500-6e93-11ea-8809-23712130a37d.png)

Reply:
@robszumski Would it be ok if we retained this wrapping behavior for longer CatalogSource names, or do you feel that is something that should be truncated?

Reply:
Wrapping seems ok. We should try to do `a-really-long-name-that-cant-truncate` and make sure we are happy with that behavior.

Reply:
Just tried this and it looked basically the same as Andy's screenshot above, which I think is fine. Will remove mention of truncation from the design.

Reply:
Awesome! Thanks!!